lib/command/createbucket.php

```diff
@@ -107,6 +107,7 @@ private function getClient() {
 		if ($cfg === null) {
 			throw new \InvalidArgumentException('No object store is configured.');
 		}
+		/* @phan-suppress-next-line PhanDeprecatedFunction */
```

Comment:
`S3Client::factory` is deprecated.
I suppressed the message from `phan` for now.